### PR TITLE
Fix screenshare icon not crossing out when a screen is shared

### LINKED ISSUE #5859: Screenshare icon does not indicate/change at all when screensharing
## Steps to reproduce
1. Start a call
2. Start a screenshare
3. Forgot you were screensharing cause it’s not obvious

### Expected behaviour
The screenshare icon communicates the state, just like audio and video on/off do. (This is also something which was the case before, although the strikethrough display might not have been the best default icon. :)

### Actual behaviour
The icon stays the exact same, not even the opacity changes.

As an example, here’s how it looks like in Discord, very present and obvious:
image

### PR DESCRIPTION
**Pull request by nickvergessen**
Fix #5859

### CODE REVIEW

File: src/components/CallView/shared/LocalMediaControls.vue [hunk 1]
```diff
@@ -81,15 +81,15 @@
 				@update:open="screenSharingMenuOpen = true"
 				@update:close="screenSharingMenuOpen = false">
 				<!-- Actions button icon -->
-				<Monitor
+				<MonitorOff
```
Comment (nickvergessen) [inline src/components/CallView/shared/LocalMediaControls.vue:84]: Ah, now I remember why this was changed.

With this patch when:
* audio is on => icon
* audio is off => icon crossed
* video is on => icon
* video is off => icon crossed
* screenshare is on => icon crossed 😕  *nickvergessen hurt itself in confussion*
* screenshare is off => icon 😕  *nickvergessen hurt itself in confussion*

So for screensharing the icon is invers to the mic/video icon and it feels odd if they are next to each others like that.
Maybe inverting the color of the background and icon is a better state representation?

Comment (PVince81) [reply src/components/CallView/shared/LocalMediaControls.vue:84]: or invert screenshare on vs off to align with the others ?

not sure what @nextcloud/designers-talk think then...

Comment (nickvergessen) [reply src/components/CallView/shared/LocalMediaControls.vue:84]: This is currently the case but not when your screen is not shared. We wouod have to use a crossed out icon for that and it feels weird...

Comment (jancborchardt) [reply src/components/CallView/shared/LocalMediaControls.vue:84]: So yeah, for audio and video the icon reflects the current state. For screensharing the logic is inverted, but I’d not necessarily say that’s a bad thing.

Here’s possibly better icons, also from Material design:
- "Present to all" https://fonts.google.com/icons?selected=Material+Icons:present_to_all
- "Cancel presentation" https://fonts.google.com/icons?selected=Material+Icons:cancel_presentation

Comment (nickvergessen) [reply src/components/CallView/shared/LocalMediaControls.vue]: Done

**Review by PVince81 — APPROVED** [review]
:+1: looks alright

sadly it seems the new icons are not in the latest vue-material-design-icons either

**Comment by PVince81** [comment]
image

image

@jancborchardt ^

**Comment by nickvergessen** [comment]
> sadly it seems the new icons are not in the latest vue-material-design-icons either

Yeah skj dived a bit into it, the lib uses another lib which wasn't updatd since january

**Pull request merged**